src/main/java/org/jenkinsci/plugins/durabletask/DurableTask.java

```diff
@@ -63,4 +65,23 @@ public void captureOutput() throws UnsupportedOperationException {
         throw new UnsupportedOperationException("Capturing of output is not implemented in " + getClass().getName());
     }
 
+    /**
+     * Requests that a specified charset be used to transcode process output.
+     * The encoding of {@link Controller#writeLog} and {@link Controller#getOutput} is then presumed to be UTF-8.
+     * If not called, no translation is performed.
+     * @param cs the character set in which process output is expected to be
+     */
+    public void charset(@Nonnull Charset cs) {
+        // by default, ignore
```

Comment:
Add some FINE/DEBUG logging to indicate that the method is not implemented?

Reply:
Could do that. In practice there are only three implementations anyway, all in this plugin.